README.md

```diff
@@ -19,6 +19,31 @@ Logic File | TC Generator File |
 ------------------ | ------------- |
 C / C++ / Java / Python | [TC Generator](/tc_generator/tc_gen.py) |
 
+## Install
+
+### How to Use ? 😃
+
+* Clone the repository `$ git clone https://github.com/NJACKWinterOfCode/HackerRank-Test-Case-Generator.git `.
```

Comment:
This is main repo, remove `NJACKWinterofCode`

Reply:
> This is main repo, remove `NJACKWinterofCode`

gave PR to NWoC directory

Reply:
No issues, use the real link anyways.

Reply:
> No issues, use the real link anyways.

@aashutoshrathi giving a new PR, closing this.